tests/Unit/NumericalAlgorithms/Spectral/Test_LegendreGauss.cpp

```diff
@@ -128,32 +128,30 @@ SPECTRE_TEST_CASE("Unit.Numerical.Spectral.LegendreGauss.DiffMatrix",
 
 namespace {
 
-void test_spectral_to_grid_points_matrix(const size_t num_points,
-                                         const Matrix& expected_matrix) {
-  const auto& matrix = Spectral::spectral_to_grid_points_matrix<
-      Spectral::Basis::Legendre, Spectral::Quadrature::Gauss>(num_points);
+void test_modal_to_nodal_matrix(const size_t num_points,
+                                const Matrix& expected_matrix) {
+  const auto& matrix =
+      Spectral::modal_to_nodal_matrix<Spectral::Basis::Legendre,
+                                      Spectral::Quadrature::Gauss>(num_points);
   CHECK_MATRIX_APPROX(expected_matrix, matrix);
 }
 
 }  // namespace
 
-SPECTRE_TEST_CASE("Unit.Numerical.Spectral.LegendreGauss.SpectralToGridPoints",
+SPECTRE_TEST_CASE("Unit.Numerical.Spectral.LegendreGauss.ModalToNodal",
                   "[NumericalAlgorithms][Spectral][Unit]") {
-  SECTION("Check 1 point") {
-    test_spectral_to_grid_points_matrix(1, Matrix(1, 1, 1.));
-  }
+  SECTION("Check 1 point") { test_modal_to_nodal_matrix(1, Matrix(1, 1, 1.)); }
```

Comment:
This isn't really a change made in this PR but would it be better to scope and `INFO` here rather than `SECTION`?

Reply:
Yes, though that should probably be done as a revamp of all these low-level tests that were written long before we decided to not use `SECTION` :)